config/routes.rb

```diff
@@ -3,6 +3,7 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root to: 'users#show'
```

Comment:
- [ ] Hi ill recommend changing the default route to `user#index` since the user index page is supposed to be the first screen. the `user#show` page is supposed to be displayed after a user has been selected. Currently, it displays an error because it's being called without a user 
<img width="1106" alt="image" src="https://user-images.githubusercontent.com/37901469/219338206-83e809c7-125c-4f83-9be7-26a5cd2798b4.png">